From 22d95d4d9ced72bdfb191bf620241994feeeeb34 Mon Sep 17 00:00:00 2001
From: Sheng Nan <b38800@freescale.com>
Date: Fri, 7 Dec 2012 17:08:07 +0800
Subject: [PATCH 07/53] ENGR00236196: mxc_vout: add YV12 format support in
 enum fmt field.

Since YV12 format is supported in mxc_v4l2_capture, it should be also
supported in mxc_vout.

Signed-off-by: Sheng Nan <b38800@freescale.com>
---
 drivers/media/video/mxc/output/mxc_vout.c |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/media/video/mxc/output/mxc_vout.c b/drivers/media/video/mxc/output/mxc_vout.c
index 388c4b3..ec789ec 100644
--- a/drivers/media/video/mxc/output/mxc_vout.c
+++ b/drivers/media/video/mxc/output/mxc_vout.c
@@ -189,6 +189,10 @@ const static struct v4l2_fmtdesc mxc_formats[] = {
 		.pixelformat = V4L2_PIX_FMT_YUV420,
 	},
 	{
+		.description = "YVU420",
+		.pixelformat = V4L2_PIX_FMT_YVU420,
+	},
+	{
 		.description = "TILED NV12P",
 		.pixelformat = IPU_PIX_FMT_TILED_NV12,
 	},
-- 
1.7.10.4

